querySQL/querySQL.Rmd

```diff
@@ -55,26 +55,28 @@ head(df)
 
 ### With JSON output
 
-```{sh, engine.path = '/bin/bash'}
-mc sql --json --query "
-    SELECT PopTotal,PopDensity FROM s3object s
-    WHERE s.Location like '%Canada%'
-    LIMIT 50
-" minio-standard/shared/blair-drummond/sql-example/TotalPopulation.parquet > Query-Data.json
-```
+<!-- ```{sh, engine.path = '/bin/bash'} -->
+<!-- #*Note: Amazon S3 Select does not support whole-object compression for Parquet objects.* -->
+<!-- #Parquet is dsabled by default since hostile crafted input can easily crash the server.  -->
+<!-- #To enable Parquet set the environment variable MINIO_API_SELECT_PARQUET=on -->
+<!-- mc-original sql --json --query " -->
+<!--     SELECT PopTotal,PopDensity FROM s3object s -->
+<!--     WHERE s.Location like '%Canada%' -->
+<!--     LIMIT 50" minio-standard/shared/blair-drummond/sql-example/TotalPopulation.parquet > Query-Data.json -->
+<!-- ``` -->
```

Comment:
Just say that you cannot run SQL queries on Parquet files for security reasons. This is a good catch though!